txn_writer.go

```diff
@@ -152,7 +152,7 @@ func (sw safeWriter) writeAllSafe(forceVendor bool) error {
 
 	if writeM {
 		if _, err := os.Stat(mpath); err == nil {
-			// move out the old one
```

Comment:
I do see instances of brief notes in stdlib as uncapitalized, period-less inline comments, like this. (They do seem to be less common than capitalized, period-having sentences, though)

There's [certainly standards for doc comments](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences) - am I missing some for inline comments, as well?